tests/CMakeLists.txt

```diff
@@ -135,4 +144,20 @@ if(AVIF_BUILD_APPS)
     add_test(NAME test_cmd COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd.sh ${CMAKE_BINARY_DIR}
                                    ${CMAKE_CURRENT_SOURCE_DIR}/data
     )
+    add_test(NAME test_cmd_lossless COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_lossless.sh ${CMAKE_BINARY_DIR}
+                                            ${CMAKE_CURRENT_SOURCE_DIR}/data
+    )
+    add_test(NAME test_cmd_metadata COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_metadata.sh ${CMAKE_BINARY_DIR}
+                                            ${CMAKE_CURRENT_SOURCE_DIR}/data
+    )
+
+    if(NOT AVIF_CODEC_AOM OR NOT AVIF_CODEC_AOM_ENCODE)
+        # Only aom encoder supports lossless encoding.
+        set_property(TEST test_cmd_lossless PROPERTY DISABLED True)
+
+        # SVT-AV1 does not support the images with odd dimensions that are used in this test.
+        set_property(TEST test_cmd_metadata PROPERTY DISABLED True)
```

Comment:
Just wondering if we can use `set_tests_properties(test_cmd_lossless test_cmd_metadata PROPERTIES DISABLED True)` here.

Did you disable the tests separately so that it's easier to give the reason why each test is disabled?

Reply:
> Just wondering if we can use `set_tests_properties(test_cmd_lossless test_cmd_metadata PROPERTIES DISABLED True)` here.

We can.

> Did you disable the tests separately so that it's easier to give the reason why each test is disabled?

Yes.